float32[3] robot_torques